src/main/java/com/booleanuk/api/cinema/utils/DataInitializer.java

```diff
@@ -0,0 +1,84 @@
+package com.booleanuk.api.cinema.utils;
+
+import com.booleanuk.api.cinema.domain.dtos.*;
+import com.booleanuk.api.cinema.services.CustomerService;
+import com.booleanuk.api.cinema.services.MovieService;
+import com.booleanuk.api.cinema.services.ScreeningService;
+import com.booleanuk.api.cinema.services.TicketService;
+import com.github.javafaker.Faker;
```

Comment:
Nice! I don't think I've used this one previously, I have used a similar library in Python though. 

If you needed to test using a Faker class then you could use it to generate a fake object that you then create the record for and check that the results match. In other words you would move the Fake part to the Tests so that you could then test using it.